modules/test/playwright/tests/calendar-web/miniCalendar.spec.ts

```diff
@@ -110,3 +115,71 @@ test('ensure that mini calendar accessibility properties are maintained after ch
 		'polite'
 	);
 });
+
+test('ensure that next and previous buttons acessibility label in the mini calendar are translated to portuguese', async ({
+	accountSettingsPage,
+	apiHelpers,
+	calendarWidgetPage,
+	page,
+	pageEditorPage,
+	site
+}) => {
+	const company =
```

Comment:
Declare all this consts (`company`, `defaultUser`, `user`) as `let` and assign inside the `try`;

Reply:
Done!